executor/include/execution/trait/is_executor.hpp

```diff
@@ -4,24 +4,31 @@
 
 #pragma once
 
+#include <execution/executor/base_executor.hpp>
 #include <experimental/type_traits>
 #include <type_traits>
 
 namespace execution {
 
-template <class Executor, typename = void>
+template <class Executor, typename Function = void (*)(), typename = void>
 struct is_executor : std::false_type {};
 
-constexpr const auto noop = [] {};
+namespace detail {
+const auto noop = [] {};
+}
 
-template <class Executor>
+template <typename Executor, typename Function>
```

Comment:
You've added the `Function` parameter, but there's no explicit use of it. That usually, means it's not required.

Reply:
You are right, it isn't required.
Removed in fe249605c8ba21fd1ea17b7132f5d246605778c6